events/validations/list.js

```diff
@@ -34,9 +34,8 @@ module.exports = [
       in: ['query'],
       toInt: true,
       custom: {
-        options(value, { req }) {
-          return !(req.query.query.beforeDate || req.query.query.afterDate) || !!value;
-        },
+        options: (value, { req }) =>
+          !(req.query.query.beforeDate || req.query.query.afterDate) || typeof value === 'number',
```

Comment:
Do we really care about the typeoff? The validator is going to cast it to Int anyway?

Reply:
The reason for using `typeof` is that `typeof 0 === 'number'` is true, while `!!0` is false. It basically just makes sure that it's not null or undefined without having to go `(value !== undefined && value !== null)`